### PR TITLE
30 write agenda invitation tests

### PR DESCRIPTION
**Pull request by pablotocalini**
I have added two tests

1. Create a new invitation
2. Create a user from an invitation: Should we test what happens when the roleId and the invitation.id are wrong?

### CODE REVIEW

File: core/test/model/agenda.test.js [hunk 1]
```diff
@@ -61,7 +62,59 @@ describe('Agenda Test', () => {
       //assert
       expect(getUserThatNotExists).toThrow(new Error(USER_NOT_FOUND))
     })
+
+    /* We need to add the unit tests for these scenarios (taking into account the errors cases)
```
Comment (crudobaker) [inline core/test/model/agenda.test.js:66]: Remove comment.

File: core/test/model/agenda.test.js [hunk 2]
```diff
@@ -61,7 +62,59 @@ describe('Agenda Test', () => {
...
+      //assert
+      expect(invitation.contactId).toBe(1)
+      expect(invitation.firstName).toBe('Pablo')
+      expect(invitation.lastName).toBe('Perez')
+    })
+
+    //create a user from an invitation (createUserFromInvitation)
+    test('create an user from invitation', () => {
+      //arrange
+      //const fakeInvitation = repository.createInvitation(1, 'Pablo', 'Perez')
```
Comment (crudobaker) [inline core/test/model/agenda.test.js:107]: Remove comment.

File: core/test/model/agenda.test.js [hunk 3]
```diff
@@ -61,7 +62,59 @@ describe('Agenda Test', () => {
...
+      const invitation = agenda.inviteUser(1, 'Pablo', 'Perez')
+      //act
+      const getUserFromInvitation = agenda.createUserFromInvitation(invitation.id, '1')
```
Comment (crudobaker) [inline core/test/model/agenda.test.js:110]: Use CoordinatorRole.id instead of just `1` value.

File: core/test/model/agenda.test.js [hunk 4]
```diff
@@ -61,7 +62,59 @@ describe('Agenda Test', () => {
...
+      //assert
+      expect(getUserFromInvitation.firstName).toBe('Pablo')
+      expect(getUserFromInvitation.lastName).toBe('Perez')
+      expect(getUserFromInvitation.role).toBe(COORDINATOR_ROLE)
+    })
   })
+  //Should we test what happens when the roleId and the invitation.id are wrong?
```
Comment (crudobaker) [inline core/test/model/agenda.test.js:117]: The business code doesn't check for a valid role. Maybe we could do it.

**Comment by crudobaker** [comment]
Good code for testing! Just a couple of comments.

File: core/test/model/agenda.test.js (hunk 3 above)
Comment (pablotocalini) [reply core/test/model/agenda.test.js:110]: done

File: core/test/model/agenda.test.js (hunk 4 above)
Comment (pablotocalini) [reply core/test/model/agenda.test.js:117]: here is what i wrote to test when a invalid role is passed

**Review by crudobaker — APPROVED** [review]

**Pull request merged**